{"description": "An issue was discovered in py-lmdb 0.97. For certain values of `md_flags`, `mdb_node_add` does not properly set up a memcpy destination, leading to an invalid write operation. NOTE: this outcome occurs when accessing a `data.mdb` file supplied by an attacker.", "methods": ["Used a vulnerability in a library (py-lmdb) to perform an invalid write operation.", "Accessed a file supplied by an attacker (data.mdb)."]}